content/blog/2024/08/2024-08-16-twim.md

```diff
@@ -252,7 +252,7 @@ Matrix SDK written in pure Dart.
 
 [td](https://matrix.to/#/@td:technodisaster.com) says
 
-> meep quick update, matrix 0.32.1 released since the last post, here's a summary:
+> meep quick update, matrix-dart-sdk 0.32.1 released since the last post, here's a summary:
```

Comment:
Hm the package is technically named "matrix" though in the dart package registry 🤔

Reply:
Yeah I still found it not great to read but I don't care enough to get held up on this.

Reply:
```suggestion
> meep quick update, matrix 0.32.1 released since the last post, here's a summary:
```
```suggestion
> meep quick update, `matrix` 0.32.1 released since the last post, here's a summary:
```